modules/subscriptions.php

```diff
@@ -1122,3 +1130,10 @@ function jetpack_do_subscription_form( $instance ) {
 	$output = ob_get_clean();
 	return $output;
 }
+
+jetpack_register_block( 'subscriptions', array( 'render_callback' => 'show_subscription_block' ) );
+
+function show_subscription_block( $attr, $content ) {
```

Comment:
I would recommend that you prefix that function to avoid potential conflicts with other plugins and themes.